core/tools/build_solana_templates

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
I did that in https://github.com/trezor/trezor-firmware/pull/3359/commits/59acc63af220d31b0af38824f954f75709e54561. But shouldn't the timestamp be set to either `programs.json` or `instructions.mako.py` depending on which is newer? (I'm not sure why this is needed so I can't tell)

Reply:
this is so that the build process knows that the generated instructions.py haven't changed since last build.
and yes, it would be a bit better to do best of `programs.json` and the `.mako` file, so that we can apply the same optimization to running `build_solana_templates` itself.

Reply:
Updated to the most recent timestamp in https://github.com/trezor/trezor-firmware/pull/3359/commits/2d77f04cf0bb67d41fc927d54b2183d18e641fb0.